Docs-update (#1829)
mui/mui-toolpad: docs/data/toolpad/data-binding.md

Field with value:

<img src="/static/toolpad/docs/data-binding/bind-6.png" alt="Binding result 2" width="902px" />

## Event handling

> Components like MUI Button have onClick event handler props. This can be binded to run a piece of code to mutate the page state.

Reviewer: ```suggestion
Some components, such as `Button`, have event handler props. Event handler props can be binded to run a piece of code to mutate the page state.
```

Reviewer 2: ```suggestion
Some components have event handler props, such as the `Button` component and its `onClick` prop. Event handler props can be bound to run a piece of code to mutate the page state.
```

> (In the example below you can see how on Click of a button, the value is copied from one component to the other)

Reviewer: ```suggestion
(In the example below you can see how after clicking the button, the value is copied from one component to the other)
```
